static/js/admin.js

```diff
@@ -1293,6 +1293,18 @@ function getAssignmentId() {
     return assignlist.options[assignlist.selectedIndex].value;
 }
 
+function assignListAlpha(assignlist) {
```

Comment:
At first I thought, why not just have the database sort these for us.  But I appreciate what you did here is better as it results in a case insensitive sort of the assignment names. -- definitely better for this purpose.